### ISSUE TITLE
[C-decker] font interface: incorrectly specifying size crashes decker

### ISSUE THREAD

**Issue by razetime** [JohnEarnest/Decker#70]
I did not read the docs correctly, and I typed this in the listener (i should have typed a pair):
```
deck.fonts.mono.size:13
```
Decker instantly crashed. This is the error I get in the end:
```
$ dkr &
$
[1]   Floating point exception/home/razetime/code/decker/c/build/decker
```

I initially found this when i tried to change the size of one of my custom fonts. the glyphs turned invisible when i changed the size to a scalar. And when i later tried to switch to it, decker crashed.

On web decker this turns the font invisible as well. I am assuming it is since this defaults to 0,0, which isn't an error, but certainly makes the listener unusable.

**Comment by JohnEarnest** [comment]
I've corrected the `font` interface to enforce a minimum font size of 1x1 pixels, which is a more reasonable default and should prevent crashes.

The fact that it is possible to mutilate built-in fonts to the point that it becomes difficult to operate Decker's UI is not a bug; it's just Decker being exciting and dynamic. It's also possible to e.g. set the `patterns` interface's color scheme to black-on-black.

In the worst case scenario you can use keyboard shortcuts "blind" to save the current document and then remove the screwed up font entries from the `.deck` file manually with your favorite text editor.

**Comment by razetime** [comment]
the workaround I did was to just click on the previous listener entry and blindly backspace and type in the value again. It's nice and funny that i can mutilate the environment and turn my cursor into a dot.

The crash is fixed in e69411f.
